README.md

```diff
@@ -5,7 +5,8 @@ Battery Monitor is a utility tool developed on Python3 and PyGtk3. It will notif
  - [Dependencies](#dependencies)
  - [Installation](#installation)
      - [Common Method](#common-method)
-     - [For Ubuntu and its derivatives](#for-ubuntu-and-its-derivatives)
+     - [For Ubuntu and it's derivatives](#for-ubuntu-and-its-derivatives)
```

Comment:
I think it should be `its derivatives`. `it's derivatives` refers to `it is derivatives`.